<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_info</class>
 <widget class="QWidget" name="add_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>278</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345</width>
    <height>278</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加学生信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>58</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>密      码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass_line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>108</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>学生密码，必填。</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>姓       名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>108</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>学生姓名，必填。</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>年      龄：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>58</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>性    别：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="age_line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>108</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>学生年龄，必填</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>学院编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>108</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>所属学院，必填。</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyan_line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>108</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>科研组号，选填。</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>科研小组组号：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveall_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>322</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="return_btn">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addInfo_btn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>学生信息增添界面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
